zap/src/main/java/org/parosproxy/paros/core/scanner/AbstractPlugin.java

```diff
@@ -604,37 +606,56 @@ private boolean isCustomPage(HttpMessage msg, CustomPage.Type cpType) {
 
     /**
      * Tells whether or not the message matches {@code CustomPage.Type.OK_200} definitions. Falls
-     * back to use {@code Analyser}.
+     * back to use {@code Analyser} which analyzes specific behavior and status codes. Checks if the
+     * message matches {@code CusotmPage.Type.ERROR_500} or {@code CusotmPage.Type.NOTFOUND_404}
```

Comment:
```suggestion
     * message matches {@code CustomPage.Type.ERROR_500} or {@code CustomPage.Type.NOTFOUND_404}
```
This typo is present multiple times below too...

Reply:
Thanks, fix inbound